### DISCUSSION TITLE
Request Fowarding

### DISCUSSION THREAD

**Discussion by MCole-K9** [sveltejs/kit#7020] [Ideas]
Not sure how this whole thing works but I was wondering if could be possible to functionally forward a Request.

Say we have our actions defined and we would like to reuse the logic in the +server.ts file. instead of creating a fetch request to which i'd assume would be (please clear this part up for me as well): 
```

await fetch(route?/namedAction, {
method: "POST",
etc.
})
```

we could in turn do 
![image](https://user-images.githubusercontent.com/81397122/192124936-112b10ce-d14e-42aa-a0c8-b7d56ea91257.png)

![image](https://user-images.githubusercontent.com/81397122/192124945-ccc3ba63-9d3e-4682-8a65-eaa9422cbd89.png)

**Comment by teemingc** [comment]
I'd recommend abstracting the your logic into a function which you use in both the `action` and `POST` request handler.

**Reply by MCole-K9** [reply]
Thank you for this reply. I thought about this as well,  and I guess it would be the logically thing to do, but I was just wondering. **I think I saw** at some point (can't seem to find it) @Rich-Harris was discussing something similar but by forwarding the request through a fetch request
